src/components/FilterForm.js

```diff
@@ -10,15 +10,33 @@ const Form = () => {
   );
 
   const [gender, setGender] = useState("M");
-  const [yearRange, setYearRange] = useState({ min: 1970, max: 1990 });
-  const [year, setYear] = useState(1900);
+  const [rangeToggle, setRangeToggle] = useState(false);
+  const [yearRange, setYearRange] = useState({ min: 1950, max: 1990 });
+  const [year, setYear] = useState(1950);
 
   const handleSubmit = () => {
-    let searchURL = `https://baby-namer-api.herokuapp.com/names?sex=${gender}&yearStart=${yearRange.min}&yearEnd=${yearRange.max}`;
+    let searchURL = `https://baby-namer-api.herokuapp.com/names?sex=${gender}&yearStart=${year}`;
```

Comment:
It would be good to have a module that contains all the logic for building urls.